packages/message-hash/src/index.ts

```diff
@@ -56,6 +56,6 @@ export function messageHashStr(
   message: IProtoMessage | IDecodedMessage
 ): string {
   const hash = messageHash(pubsubTopic, message);
-  const hashStr = bytesToUtf8(hash);
+  const hashStr = bytesToHex(hash);
```

Comment:
this is fix for `messageHashStr` when it doesn't transform bytes properly and shows weird symbols